frontend/src/app/shared/shared.module.ts

```diff
@@ -38,6 +38,7 @@ import { SplitButtonModule } from 'primeng/splitbutton';
 import { PasswordModule } from 'primeng/password';
 import { TabViewModule } from 'primeng/tabview';
 import { ButtonModule } from 'primeng/button';
+import { BoxPropButtonComponent } from './box-components/box-prop-button/box-prop-button.component';
```

Comment:
Place on line 27, next to the components

Reply:
Done